deb/openmediavault/usr/share/openmediavault/engined/rpc/certificatemgmt.inc

```diff
@@ -218,19 +218,46 @@ class CertificateMgmt extends \OMV\Rpc\ServiceAbstract {
 		// that the certificate has not been created or updated until now.
 		if ($this->isModuleDirty("certificates"))
 			throw new \OMV\Config\ConfigDirtyException();
+		$output = [];
+		// Private key path.
+		$keyPath = sprintf(
+			"%s/private/%s%s.key",
+			\OMV\Environment::get("OMV_SSL_CERTIFICATE_DIR"),
+			\OMV\Environment::get("OMV_SSL_CERTIFICATE_PREFIX"),
+			$params['uuid'],
+		);
+		$output[] = sprintf("Private Key Path: %s", $keyPath);
+		// Certificate path.
+		$certPath = sprintf(
+			"%s/certs/%s%s.crt",
+			\OMV\Environment::get("OMV_SSL_CERTIFICATE_DIR"),
+			\OMV\Environment::get("OMV_SSL_CERTIFICATE_PREFIX"),
+			$params['uuid'],
+		);
+		$output[] = sprintf("Certificate Path: %s", $certPath);
```

Comment:
I do not like to show this information in the UI. This is against the philosophy of OMV which is to hide sensitive data in the UI. Please remove that.